bigtop-packages/src/common/ignite-hadoop/do-component-build

```diff
@@ -21,4 +21,23 @@ set -ex
 export MAVEN_OPTS="-Xmx512M"
 
 sed -i '/asm-all/{n;s/4.2/5.0.2/}' modules/hadoop/pom.xml
+
+# download ignite-shmem from maven central
+mvn dependency:get -DremoteRepositories=https://repo1.maven.org/maven2/ -DgroupId=org.gridgain -DartifactId=ignite-shmem -Dversion=1.0.0
```

Comment:
I do not think we need this at all since we replace the only necessary file anyway.